subprojects/packagefiles/openssl/meson.build

```diff
@@ -0,0 +1,406 @@
+project(
+  'openssl',
+  'c',
+  version: '1.1.1k',
+  license : 'Apache-2.0',
+  meson_version: '>= 0.55',
+  default_options: [
+    'warning_level=1',
```

Comment:
OpenSSL sets `-O3` by default. Do we want to do the same by setting `optimization=3` here?

Reply:
Or maybe the default `buildtype` should be `release`?

Reply:
OpenSSL does use `-O3`, but since Meson allows these to be configured during setup process I decided that it is probably better to not specify here any level and allow Meson to configure it. In case of release build it will be set to `-O3` automatically. `-O3` is removed from automatically generated compiler flags (from OpenSSL's build system) so Meson is in full control of it.

None of the other wraps seem to specify optimization level unless I'm missing something, that is the primary rationale for not specifying optimization level or build type (I'm not even sure `buildtype` applies to individual subprojects).

Reply:
Setting the default buildtype is usually a good thing for a project, but let's leave it as-is for now since it can turn into a policy question around wraps.